src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -791,31 +821,52 @@ def collect_periscope_logs(resource_group_name, name, storage_account_name=None,
 
             print()
             print(f"{colorama.Fore.GREEN}Deploying diagnostic container on the K8s cluster...")
-            subprocess_cmd = kubectl_prior + ["apply", "-f", temp_yaml_path, "-n", "aks-periscope"]
+            subprocess_cmd = kubectl_prior + ["apply", "-f", temp_yaml_path, "-n", "aks-periscope", "--validate=false"]
             subprocess.check_output(subprocess_cmd, stderr=subprocess.STDOUT)
         except subprocess.CalledProcessError as err:
             raise CLIInternalError(err.output)
     finally:
         os.remove(temp_yaml_path)
 
-    print()
-    # log_storage_account_url = f"https://{storage_account_name}.blob.core.windows.net/"
 
-    print(f'{colorama.Fore.GREEN}Your logs are being uploaded to storage account {format_bright(storage_account_name)}...')
-
-    print()
-    print(f'You can download Azure Storage Explorer here '
-          f'{format_hyperlink("https://azure.microsoft.com/en-us/features/storage-explorer/")}'
-          f' to check the logs by accessing the storage account {storage_account_name}.')
-    # f' to check the logs by adding the storage account using the following URL:')
-    # print(f'{format_hyperlink(log_storage_account_url)}')
+def copy_and_zip_periscope_files(kubectl_prior):
+    periscope_files = []
+    time.sleep(5)   
+    subprocess_cmd = kubectl_prior + ["get", "pods", "-n", "aks-periscope", "-o", "wide", "--no-headers"]
+    subprocess.call(subprocess_cmd, stderr=subprocess.STDOUT)
+    pods = subprocess.check_output(
+        subprocess_cmd,
+        universal_newlines=True)
+    pod_lines = pods.splitlines()
+    for line in pod_lines:
+        time.sleep(5)
```

Comment:
Have we intentionally kept time sleep inside loop(for each pod). We can keep it outside the loop?

Reply:
Ok, I can increase the length of time sleep outside of the loop and see what happens.